internal/chrome_desktop.py

```diff
@@ -25,7 +25,8 @@
     '--disable-default-apps',
     '--disable-domain-reliability',
     '--safebrowsing-disable-auto-update',
-    '--disable-background-timer-throttling'
+    '--disable-background-timer-throttling',
+    '--no-sandbox'
```

Comment:
Makes sense. I would add a "--dockerized" option, as we also might need to change some other bits to run in docker (maybe traffic shaping).
While `--no-sandbox` isn't a good default choice, it shouldn't be problematic inside docker, as the whole container is sandboxed.